lib/cnet/pcb/cnet_pcb.c

```diff
@@ -203,29 +180,41 @@ pcb_show(mempool_t *mp __cne_unused, void *obj_cb_arg __cne_unused, void *obj,
                  ntohs(CIN_PORT(&pcb->key.laddr))) < 0)
         CNE_RET("Truncated buffer data\n");
 
-    cne_printf(" [cyan]%20s[]\n", lbuf);
+    cne_printf(" [cyan]%20s[] %4d\n", lbuf, pcb->ttl);
 }
 
 static void
-_pcb_dump(stk_t *stk, int type)
+__pcb_iterate(mempool_t *mp __cne_unused, void *obj_cb_arg __cne_unused, void *obj,
+              unsigned n __cne_unused)
+{
+    __pcb_show((struct pcb_entry *)obj);
+}
+
+static void
+_pcb_dump(stk_t *stk)
```

Comment:
Why do you have a leading underscore here?

Reply:
Removed the underscores.